benchmark/attri_util.py

```diff
@@ -0,0 +1,132 @@
+import operator
+from dataclasses import dataclass
+from enum import Enum
+from functools import reduce
+from typing import Any, List, Optional, Tuple
+
+
+class ReadOnly:
+    def __init__(self, value):
+        self._value = value
+
+    @property
+    def value(self):
+        return self._value
+
+
+BLAS_OPS = ReadOnly(["addmm", "mv", "addmm", "mm", "outer"])
+
+DEFAULT_WARMUP_COUNT = 100
+DEFAULT_ITER_COUNT = 1000
+
+# BLAS situation
+# BLAS shapes is defined by (B,M,N,K), it is different from the non blas Shapes
+DEFAULT_BLAS_BENCH_SHAPES = [(1, 1, 1, 32), (4, 15, 160, 1024), (16, 495, 5333, 71)]
+DEFAULT_BLAS_WITHOUT_BATCH_BENCH_SHAPES = [(1, 1, 32), (15, 160, 1024), (495, 5333, 71)]
```

Comment:
What about BLAS_DEFAULT_BMNK, BLAS_DEFAULT_MNK?
Also, the larger sizes are rather small.